src/types/resolveConstantValue.ts

```diff
@@ -121,6 +121,23 @@ function reduceAddress(ast: ASTExpression, ctx: CompilerContext): Address {
                 }
                 return address;
             }
+        } else if (ast.name === "newAddress") {
+            if (ast.args.length === 2) {
+                const wc = reduceInt(ast.args[0]);
+                const addr = Buffer.from(
+                    reduceInt(ast.args[1]).toString(16),
+                    "hex",
+                );
+                if (!enabledMasterchain(ctx)) {
```

Comment:
We are not checking the workchain is either `-1` or `0` like in the code above. Why is that?